src/utils/args.ts

```diff
@@ -258,12 +259,10 @@ class ArgsError {
 
     validOrNull(value: any) {
 
-        let valid = true;
+        let valid = false;
         try {
-            if (!this.validator(value))
-                valid = false;
+            if (this.validator(value)) valid = true;
         } catch(e) {
-            valid = false;
             this.intermediate = value.value;
         }
```

Comment:
Not complaining, but curious - why?

Reply:
No bug fixes here. Just cleaner to set `true` once instead of setting `false` twice.